### PR TITLE
GumGum: adds us privacy (CCPA) support

### PR DESCRIPTION
**Pull request by susyt**
#3923  Type of change

- [X] Feature

## Description of change

adds support for us privacy/ccpa in adapter

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/
https://github.com/prebid/prebid.github.io/pull/1673

## Other information

cc @derberbaby @WayneYang1

### CODE REVIEW

**Comment by bretg** [comment]
docs pr https://github.com/prebid/prebid.github.io/pull/1673

File: modules/gumgumBidAdapter.js [hunk 1]
```diff
@@ -198,6 +199,9 @@ function buildRequests (validBidRequests, bidderRequest) {
     if (data.gdprApplies) {
       data.gdprConsent = gdprConsent.consentString;
     }
+    if (uspConsent) {
+      data.uspConsent = uspConsent;
```
Comment (harpere) [inline modules/gumgumBidAdapter.js]: the param should be `us_privacy`

**Comment by harpere** [comment]
docs: https://github.com/prebid/prebid.github.io/pull/1673/files

**Comment by susyt** [comment]
hey @harpere 
are you referring to the getUserSyncs method? If so, GumGum's ad server returns the userSync pixels with the consent appended - no change is needed
As for the incorrect param, it's been requested by our internal team to name the us privacy consent parameter as uspConsent - is it a hard requirement to name it us_privacy?

**Comment by susyt** [comment]
hi @harpere & @bretg ,
is there anything else holding this PR back from approval?

File: modules/gumgumBidAdapter.js (hunk 1 above)
Comment (harpere) [reply modules/gumgumBidAdapter.js]: actually, although the IAB spec for USP/CCPA suggests `us_privacy` as the industry standard, you're not required to use that parameter for your endpoint.

**Comment by harpere** [comment]
no, the param name is not a hard requirement.

**Comment by harpere** [comment]
yes, referring to the `getUserSyncs()` method.  OK if it's already appended.

**Pull request merged**